.github/workflows/tests.yml

```diff
@@ -10,7 +10,6 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GOPROXY: "https://proxy.golang.org,direct"
-      ENABLE_DOCKER_INTEGRATION_TESTS: TRUE
```

Comment:
This is no longer needed since `//go:build integration` is used in `integration_test.go`